app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -32,4 +33,20 @@ def configuration_scripts
       o[:inventory_root_group] = target.inventory_groups.lazy_find(i.inventory_id.to_s)
     end
   end
+
+  def configuration_script_sources
+    collector.projects.each do |i|
+      o = target.configuration_script_sources.find_or_build(i.id.to_s)
+      o[:description] = i.description
+      o[:name] = i.name
+
+      i.playbooks.each do |playbook_name|
+        # FIXME: its not really nice how I have to build a manager_ref / uuid here
+        p = target.playbooks.find_or_build("#{i.id}__#{playbook_name}")
```

Comment:
> so it's better to have in unchanged in our db

what should I set it to then? If I do this
```
target.playbooks.find_or_build(playbook_name)
```
the tests fail, because there are duplicate playbook names across projects